service/request/index.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+import got from 'got';
+import merge from 'lodash.merge';
+
+function createRequestService() {
+    function post(options) {
+        let opts = {
+            method: 'POST',
+            headers: {
+                accept: 'application/vnd.api+json',
+                'Content-Type': 'application/vnd.api+json'
+            },
+            responseType: 'json',
+            throwHttpErrors: false
+        };
+        // allow external overriding of the default internal opts
+        // for example to change responseType, or headers
+        opts = merge(opts, options);
+        return got(opts);
```

Comment:
Error handling / instrumentation

Current design returns the raw got promise; no wrapper for logging, timing, retry, or circuit-breaker concerns.
Consider optionally injecting a logger or metrics handler (e.g., latency, status codes).